pkg/tlog/tlog.go

```diff
@@ -0,0 +1,24 @@
+package tlog
+
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+import (
+	"testing"
+)
+
+//Testing implements testing interface
```

Comment:
does it? I thought it implemented log.Logger?

Reply:
It implements the Print and Printf which are required to satisfy the log.Logger interface.Hence it does implement it indirectly.